front/src/applications/stdcmV2/components/StdcmVias.tsx

```diff
@@ -27,10 +29,19 @@ const StdcmVias = ({ disabled = false, setCurrentSimulationInputs }: StdcmConfig
     useOsrdConfActions() as StdcmConfSliceActions;
   const pathSteps = useSelector(getPathSteps);
 
+  const [stopTypes, setStopTypes] = useState<StdcmStopTypes[]>(
+    pathSteps.map(() => StdcmStopTypes.PASSAGE_TIME)
+  );
```

Comment:
It might be safer to group them by pathStep ids instead of their indexes in case one of the via is removed ? What do you think ?

```suggestion
  const [stopTypes, setStopTypes] = useState<Record<string, StdcmStopTypes>>(
    compact(pathSteps).reduce(
      (acc, cur) => {
        acc[cur.id] = StdcmStopTypes.PASSAGE_TIME;
        return acc;
      },
      {} as Record<string, StdcmStopTypes>
    )
  );
```

Reply:
I agree that it would be safer, but the stop time is not updated when the page is reloaded with this modification and stop time input appears for the passage stop type